release-notes/opensearch-dashboards-reports.release-notes-2.0.0.0-rc1.md

```diff
@@ -0,0 +1,25 @@
+## Version 2.0.0.0-rc1 Release Notes
+
+Compatible with OpenSearch 2.0.0.0-rc1
+
+### Bug Fixes
+* Bump async from 3.2.0 to 3.2.3 in /dashboards-reports ([#338](https://github.com/opensearch-project/dashboards-reports/pull/338))
+* Bump moment from 2.29.1 to 2.29.3 in /dashboards-reports ([#344](https://github.com/opensearch-project/dashboards-reports/pull/344))
+* Bump minimist from 1.2.5 to 1.2.6 in /dashboards-reports ([#321](https://github.com/opensearch-project/dashboards-reports/pull/321))
+
+### Infrastructure
+* Added missing zip for bwc tests ([#329](https://github.com/opensearch-project/dashboards-reports/pull/329))
+* Remove JDK14 from CI ([#335](https://github.com/opensearch-project/dashboards-reports/pull/335))
+* Updated issue templates from .github. ([#328](https://github.com/opensearch-project/dashboards-reports/pull/328))
+* Remove zips and download from remote at build time ([#337](https://github.com/opensearch-project/dashboards-reports/pull/337))
```

Comment:
Any definition of what infrastructure means here. I have included them in Maintenance for sql

Reply:
> Infrastructure
   A change to infrastructure, testing, CI/CD, pipelines, etc.

> Maintenance
  A  change to add support for new versions of ES/Kibana from upstream